docs/src/modules/spring/pages/views.adoc

```diff
@@ -106,15 +106,14 @@ You can create a View from an Event Sourced Entity by using events that the Enti
 
 This example assumes a Customer equal to the previous example and an Event Sourced Entity that uses this Customer, and it is in charge of producing the events that update the View. These events are defined as subtypes of the class `CustomerEvent` following standard https://github.com/FasterXML/jackson-annotations#handling-polymorphic-types[Jackson notation] like this:
 
-.src/main/java/customer/api/CustomerEvent.java
+.src/main/java/customer/domain/CustomerEvent.java
 [source,java]
 ----
 include::example$spring-eventsourced-customer-registry/src/main/java/customer/domain/CustomerEvent.java[tags=class]
 ----
-<1> Includes the logical type name (defined in implementation classes) as type 
-<2> Required for deserialization and to define the logical type name 
+<1> Includes the logical type name using `@TypeName` annotation.
 
-Adding `@JsonTypeInfo` is needed so that Jackson can read back correctly the Object type when deserializing. By using `name`, you set its value as the fully qualified message. This is called a `logical type name` and it allows you to refactor the class to another package after you have serialized messages of this type. Otherwise, after such a refactor the objects serialized with the old package won't be recognized as the same type. 
+IMPORTANT: It's highly recommended to add a `@TypeName` to your persisted events. Kalix needs to identify each event in order to deliver them to the right event handlers. If a logical type name isn't specified, Kalix will use the fully qualified class name. However, once a fully qualified class name is persisted in Kalix, you won't be able to rename your class or to change its package, as Kalix won't be able to recognize previously persisted event. Therefore, we recommend all event classes to use a logical type name.
```

Comment:
The question is, should we highly recommend or make it compulsory?

Reply:
Could we through some warnings at compilation?

Reply:
If we have a warning, it will be at runtime, when we scan the components to build the descriptor. 

A warning will annoy the user. If one day they decide to add the annotation, they will break the system. Unless they use the FQCN as value, because a running system will have already events persisted with the FQCN.

Reply:
I see. Too late then. Okay

Reply:
So, I change this to use the Simple Class Name in absence of a TypeName. That allows users to still change the package and it follows what Jackson does as well. 

Users are not forced to add it, so for someone that is starting with Kalix, that's one less thing to take into account. One less thing to demo and explain for us as well. 

Eventually, users will need to add the TypeName later. Also, we could solve it when we introduce schema migration, as discussed in: https://github.com/lightbend/kalix/issues/7953

Reply:
Agreed. Great to have the issue created so we don't forget.